public/scripts/main.js

```diff
@@ -253,7 +253,7 @@ function updateTimingDisplay() {
 
 function onIncorrect() {
   $('#textdiv').effect("shake", { distance: 3 });
-  $("#read").addClass('incorrect');
+  document.querySelector("#read").classList.add('incorrect');
   setTimeout(clearPressedKeys, 500);
 };
 
```

Comment:
Please resolve the conflicts and also if possible reference the issue in the header.

Reply:
@apoorva-mk Done!